### ISSUE TITLE
Errors when going through the examples notebook

### ISSUE THREAD

**Issue by lassekp** [polca/pathways#4]
[Pathways errors.docx](https://github.com/user-attachments/files/17540605/Pathways.errors.docx)

Hello,
I tried going through the examples notebook this morning, and encountered some errors (see attached). I did not change any of the code in the notebook.

Any fixes to these issues?

Thanks in advance!
- Lasse Poulsen

**Comment by romainsacchi** [comment]
Hi @lassekp,

1. this is only a warning because one field of the data package descriptor does not conform to the validation rules. Although annoying, it can be ignored.
2. Right. This is because the class `PathwaysDataPackage` is not just yet in the stable release/main branch of `premise`. To use it, you have to install the `pathways` branch of `premise`, like so: `pip install git+https://github.com/polca/premise.git@pathways` This will install that specific version of premise which contains that class, which allows generating `pathways`-compatible data packages containing `premise` databases inside.
3. Indeed, for some reason, that `double_counting` argument disappeared. We're working on re-implementing this now. I will let know you when that happens.

**Comment by lassekp** [comment]
Thanks @romainsacchi.

I installed the specific Premise version, which fixed the issue - However, now I get a new error when trying to create a database (attached). I dont have this specific excel-file that it is searching for, and I can't seem to find it on the Premise github either. Should it be looking for the yml file instead?
![Errno 2 No such file or directory](https://github.com/user-attachments/assets/6dddfe34-2e80-4b7f-aff0-8bb57662ae01)
![metals folder](https://github.com/user-attachments/assets/8e7f6125-1a72-4aef-bb21-de29b9dc3607)

**Comment by romainsacchi** [comment]
Indeed, it seems I forgot to declare it in the packaging setup. As a quick fix, you can download the missing files from [here](https://github.com/polca/premise/tree/pathways/premise/data/metals) and paste them into your folder.

**Comment by romainsacchi** [comment]
This issue should no longer exist (with `1.0.1`). I'll close it, but feel free to reopen it if needed.
